mod-pubsub-server/src/main/java/org/folio/services/impl/SecurityManagerImpl.java

```diff
@@ -206,4 +219,9 @@ public SecurityManagerImpl(@Autowired PubSubUserDao pubSubUserDao) {
     return permissions;
   }
 
+  private Boolean putTokenToVertxContext(String token, OkapiConnectionParams params) {
```

Comment:
You never used return value. So, pls remove it. Moreover, there will be better to use primitive "boolean" (if needed).

Reply:
done